pyproject.toml

```diff
@@ -43,21 +43,21 @@ tutorial = [
     "matplotlib",
     "h5netcdf",
 ]
-maintainer = ["build"]
+maintainer = ["build", "hatch"]
```

Comment:
@tennlee, looks like I had the foresight to add the dependency already. Ready for review now